docs/pages/installation.mdx

```diff
@@ -179,12 +168,40 @@ We also provide the following images for FIPS builds of Teleport Enterprise:
 For testing, we always recommend that you use the latest release version of
 Teleport Enterprise, which is currently `(=teleport.latest_ent_docker_image=)`.
 
-[Ubuntu 20.04](https://hub.docker.com/\_/ubuntu)-based images for non-FIPS
-Teleport are available from our [Legacy Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent).
-
 </TabItem>
 </Tabs>
 
+#### Interacting with Distroless Images
+
+Since version 15, Teleport images are based on Google's [Distroless](https://github.com/GoogleContainerTools/distroless) images.
+Those images don't contain any shell. Here are some on how to invoke commands in shell-less images if you are unfamiliar:
```

Comment:
```suggestion
Those images don't contain any shell. 

To execute Teleport commands on containers based on these images, run commands similar to the following:
```
- Missing word after "Here are some..."
- "If you are unfamiliar" could apply to any docs instruction